src/test/java/uk/co/ramp/covid/simulation/place/ShopTest.java

```diff
@@ -38,8 +38,8 @@ public void initialise() throws JsonParseException, IOException {
     }
 
     @Test
-    public void testShopTransProb() {
-        double expProb = PopulationParameters.get().getpBaseTrans() * 5d / (5000d / 200d);
+    public void testShopTransProb() throws JsonParseException {
```

Comment:
I'm not sure this throws a JsonParseException? since the parameters should be read elsewhere?